change.go

```diff
@@ -39,52 +39,34 @@ const (
 	ChangeStatusCommitted ChangeStatus = "committed"
 )
 
-// Change in the system state.
+// Change to a single configuration bundle for a tag or a node.
```

Comment:
OK, I see the problem here. I think we shouldn't change the existing data structure, as there might be integration depending on it. Instead, I would suggest we introduce a new type `ChangeRequest` which would be used as input to the `CreateConfigurationChange`, then we can use the existing `Change` type as the output of that method.

Please let me know if we should discuss that on a video call.